test/e2e/specs/specs-playwright/wp-likes__post-spec.js

```diff
@@ -71,15 +71,21 @@ describe( DataHelper.createSuiteTitle( 'Likes (Post)' ), function () {
 			await BrowserManager.clearAuthenticationState( page );
 		} );
 
-		it( `Like post as ${ anotherUser }`, async function () {
+		it( `Like post as ${ likeUser }`, async function () {
 			publishedPostPage = new PublishedPostPage( page );
 
-			const loginFlow = new LoginFlow( page, anotherUser );
-
 			// Clicking the Like button will bring up a new popup.
-			// `loginFromPopup` will observe for a popup event, grab the new popup and
-			// execute the login process on that page.
-			await Promise.all( [ loginFlow.logInFromPopup(), publishedPostPage.likePost() ] );
+			// Observe for the new popup in the Promise. Once the Page object
+			// is obtained, execute login steps there and wait for the
+			// popup to close.
+			await Promise.all( [
+				page.on( 'popup', async ( popup ) => {
+					const loginPage = new LoginPage( popup );
+					await loginPage.login( { account: likeUser } );
+					await popup.waitForEvent( 'close' );
+				} ),
+				publishedPostPage.likePost(),
```

Comment:
I think the only noteworthy part of the PR is here.

Previously, there was a method `loginFromPopup` that would act on the Page object of the popup page. Now that `LoginFlow` is being removed, I have decided against implementing a similar method in `LoginPage`.

It felt unwise to have `LoginPage` accept one instance of a page when being constructed, but work its methods on another (a là `LoginFlow`). Since the instances of logging into Calypso via a popup is exactly 1 in the entire suite, I decided it was better to handle popups on a case-by-case basis.